src/schemas/json/io-package.json

```diff
@@ -0,0 +1,504 @@
+          "type": "string",
+          "pattern": "(?i)^(?!iobroker)"
+        },
+        "version": {
+          "description": "Current adapter version",
+          "type": "string",
+          "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+        },
+        "platform": {
+          "description": "Platform, normally Javascript/Node.js",
+          "type": "string",
+          "enum": ["Javascript/Node.js"]
+        },
+        "titleLang": {
+          "description": "Longer name of adapter in all supported languages like {en: 'Adapter', de: 'adapter', ru: 'Драйвер'}",
+          "$ref": "#/definitions/multilingual"
+        },
+        "news": {
+          "description": "Short form of changelog in all supported languages",
+          "type": "object",
+          "minProperties": 1,
+          "maxProperties": 20,
+          "patternProperties": {
+            "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$": {
+              "$ref": "#/definitions/multilingual"
+            }
+          },
+          "additionalProperties": false
+        },
+        "desc": {
+          "description": "Description, object for multilingual description",
+          "$ref": "#/definitions/multilingual"
+        },
+        "messagebox": {
+          "description": "true if messagebox supported. If yes, the object system.adapter.<instance>.messagebox will be created to send messages to adapter",
+          "type": "boolean"
+        },
+        "docs": {
+          "description": "The structure like {\"en\": \"docs/en/README.md\", \"de\": [\"docs/de/README.md\", \"docs/de/README1.md\"]} that describes the documentation if not in README.md",
+          "type": "object",
+          "required": ["en"],
+          "patternProperties": {
+            "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+              "type": ["string", "array"],
+              "minItems": 1,
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "additionalProperties": false
+        },
+        "authors": {
+          "description": "Array of authors as string",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "string"
+          }
+        },
+        "license": {
+          "description": "License of the software",
+          "type": "string",
+          "pattern": "(?i)afl-3.0|apache-2.0|artistic-2.0|bsl-1.0|bsd-2-clause|bsd-3-clause|bsd-3-clause-clear|cc|cc0-1.0|cc-by-4.0|cc-by-sa-4.0|wtfpl|ecl-2.0|epl-1.0|epl-2.0|eupl-1.1|agpl-3.0|gpl|gpl-2.0|gpl-3.0|lgpl|lgpl-2.1|lgpl-3.0|isc|lppl-1.3c|ms-pl|mit|mpl-2.0|osl-3.0|postgresql|ofl-1.1|ncsa|unlicense|zlib"
```

Comment:
This is the other case where we want to be case insensitive, which is very ugly without using `(?i)`